docs/adr/0031-process-evolution.md

```diff
@@ -0,0 +1,171 @@
+- Expects minimal changes to strings once available
+- Expects strings available with enough lead time to localize them before a
+  major feature ships. We cannot ship major features which aren’t localized.
+
+#### Relying parties
+
+- Expects a stable platform and high availability
+
+## Considerations
+
+### Do we need to push all 8 servers all the time?
+
+If we allow the servers to run different versions of code we need to be
+mindful of:
+
+- Careful API versioning to ensure the servers can continue to communicate with each other
+- Careful database updates if more than one server is accessing the database directly
+- Differences between staging and production when testing
+- An easy way to know what version each server is running
+
+In light of the above it’s probably best to continue to keep all servers pushed
+to their latest versions. _That doesn’t necessarily mean the same version_, but
+if a newer version exists it should go out.
+
+### Do we need tags?
+
+When we are ready to deploy our code we tag it (eg. v1.100.0) as an easy means
+of identifying the code we are pushing. Tags are just aliases in git so our
+usage is purely for ease of use. That said, it is significantly easier to
+remember a tag name than a hash which makes it easier to identify what is live,
+what was live at an arbitrary time in the past, and if we need to roll back,
+easier to get the right code live. Since they offer benefits and no real
+drawback we should continue to use tags.
+
+### Who uses the change logs and what are their expectations?
+
+We’ve maintained change logs in each package for many years and they are
+updated with each release. We release often enough that not all of the
+packages change between each release and we regularly update the changelogs
+just to say nothing has changed. This causes considerable churn in our
+deployment pipeline for no reason.
+
+We asked our stakeholders about the changelogs and the only people currently
+using them are QA. After [a
+conversation](https://groups.google.com/a/mozilla.com/g/fxa-staff/c/OuKEsHs67fk)
+we found an alternative that would allow us to stop writing our change logs to
+the repository and reduce our deployment churn.
+
+### Do we need to maintain versions in each package.json?
+
+Our build script currently increments the version of every package in our
+monorepo when we run it. But do we need those versions? [The docs
+say](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#version) “If
+you don't plan to publish your package, the name and version fields are
+optional.” We used to publish our packages to npm but we don’t do that anymore.
+SRE doesn't use the version field in any scripts. We use the version to [tell
+what is live](https://accounts.firefox.com/__version__) but the commit hash is
+also there and we could look at other options.
+
+So, no, we don't need the version in each package.json.
+
+## Decision Outcome
+
+TBD, but the authors are recommending option 3
+
+## Options
+
```

Comment:
Could we add an option like "keep things as-is" to highlight a more direct comparison of what the current release script does for us, our current branching strategy, and the process involved for ROs vs the proposed options (or at least more detail regarding RO expectations for option 3)? I know we linked to the release owner doc from ecosystem-platform but it'd be nice to have a TLDR here and I'm sure we'll change that page in the future.

Also, the options cover branching strategies, but I don't see mention of using Github releases (maybe it's implied through the changelog section?) over our current automated release script or what an altered release script could look like.